### ISSUE TITLE
Feature request: Prioritize failed connector tasks

### ISSUE THREAD

**Issue by cobolbaby** [redpanda-data/console#400]
![1023567009](https://user-images.githubusercontent.com/11460848/175500194-b9c4216d-dda8-4af8-8de1-d1aaed7998b7.jpg)

When there are many connectors, it is not convenient for users to locate the problem task because it cannot be filtered by the task status.

**Comment by weeco** [comment]
Hey, would it help to allow filtering / sorting by failed tasks?

**Comment by cobolbaby** [comment]
> Hey, would it help to allow filtering / sorting by failed tasks?

Yes, it's a solution.

**Comment by twmb** [comment]
@ivpanda do we have UI mockups for what this should look like?

**Comment by ivpanda** [comment]
No, this is my first time seeing this. From what I understand, the problem is two-fold: 

1. It's not easy to visually differentiate a failed task (UI design) 
2. It's not possible to see failed tasks together (filtering) 

Is that accurate? If so, I can take a look at this.
